modules/01-mods.js

```diff
@@ -403,9 +409,9 @@ async function slashModPurge(interaction) {
       .setColor(0x00ff00)
     ] });
 
-    await interaction.followUp({ content: `${number - num} messages deleted.`, ephemeral: true });
+    await interaction.editReply({ content: `${number - num} messages deleted.` });
   } else {
-    await interaction.editReply({ content: `You need to tell me how many to delete!`, ephemeral: true });
```

Comment:
The approach to `editReply` and then `followUp` was taken in case of large purges, as (depending on how much `bulkDelete` handles) the purge may be very slow. Did you find issues with that approach during testing? If not, it should be restored.